### ISSUE TITLE
Landing Page Summarizing the Demo Apps

### ISSUE THREAD

**Issue by bradley-holt** [ibm-watson-data-lab/shopping-list#93]

**Comment by ptitzler** [comment]
According to the documentation https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/ we cannot use a branch other than master:
 > If your site is a User or Organization Page that has a repository named <username>.github.io or <orgname>.github.io, you cannot publish your site's source files from different locations. User and Organization Pages that have this type of repository name are only published from the master branch.

**Comment by rajrsingh** [comment]
I think you're misunderstanding the docs @ptitzler (I went back over it multiple times too). They mean if your repo name is <something>.github.io, it has to use the master branch for GH publication. This repo's name is `shopping-list` so that rule doesn't apply.

**Comment by bradley-holt** [comment]
@ptitzler What @rajrsingh said. 😄

You should be able to create new orphan `gh-pages` branch with something like the following:

    git checkout --orphan gh-pages
    git rm -rf .
    touch index.html
    git add index.html
    git commit -m "Make index"
    git push -u origin gh-pages

**Comment by ptitzler** [comment]
Yup, my interpretation wasn't correct. I tested this in my own org.

**Comment by ptitzler** [comment]
Catalogued the following demo app implementations:
 * Electron and PouchDB https://github.com/ibm-watson-data-lab/shopping-list-electron-pouchdb
 * Cordova and PouchDB https://github.com/ibm-watson-data-lab/shopping-list-cordova-pouchdb
 * Ionic and PouchDB https://github.com/ibm-watson-data-lab/shopping-list-ionic-pouchdb
 * React Native and PouchDB  https://github.com/ibm-watson-data-lab/shopping-list-react-native-pouchdb
 * Kotlin and Cloudant Sync https://github.com/ibm-watson-data-lab/shopping-list-kotlin-cloudant-sync

**Comment by rajrsingh** [comment]
@ptitzler can PWAs be the first/default tab on the page?

**Comment by bradley-holt** [comment]
@ptitzler Following on the comment from @rajrsingh, the ideal tab order would be:

1. Progressive Web Apps
2. Hybrid Mobile Apps
3. Native Mobile Apps
4. Desktop Apps

**Comment by ptitzler** [comment]
* Tab order was changed. 
* About now links to the https://github.com/ibm-watson-data-lab/shopping-list repo

**Comment by vabarbosa** [comment]
@ptitzler initial commit of vanilla js tutorial: https://github.com/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb/tree/master/tutorial

**Comment by ptitzler** [comment]
[Metadata was added](https://github.com/ibm-watson-data-lab/shopping-list/commit/76851b8ea71a58629482d54172e9ccd65621773b) @vabarbosa

**Comment by vabarbosa** [comment]
@ptitzler initial draft for electron tutorial: https://github.com/ibm-watson-data-lab/shopping-list-electron-pouchdb/tree/master/tutorial
